corehq/apps/app_manager/tests/data/list_apps/list_apps.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <apps>
-  <app profile="http://onering.com/a/rohan/apps/download/app_id/profile.xml?latest=true" domain="rohan" name="Untitled Application" environment="production" media-profile="http://onering.com/a/rohan/apps/download/app_id/media_profile.xml?latest=true"/>
```

Comment:
the `app_id` _is_ the version. 
We create a new `Application` document in the database whenever we make a build which is a "copy of" the actual `Application` at that point in time. This new document has its own `app_id`, which corresponds to a particular version of the app.